language/stdlib/modules/DualAttestationLimit.move

```diff
@@ -0,0 +1,50 @@
+address 0x0 {
+
+module DualAttestationLimit {
+    use 0x0::LibraConfig;
+    use 0x0::Signer;
+    use 0x0::Transaction;
+    use 0x0::Association;
+
+    struct DualAttestationLimit {
+        micro_lbr_limit: u64,
+    }
+
+    // A wrapped capability to allow updating of the on-chain config by Blessed
+    resource struct ModifyCapability {
+        cap: LibraConfig::ModifyConfigCapability<Self::DualAttestationLimit>,
+    }
+
+    /// Travel rule limit set during genesis
+    public fun initialize(account: &signer, tc_account: &signer) {
+        Transaction::assert(Signer::address_of(account) == LibraConfig::default_config_address(), 1);
+        let cap = LibraConfig::publish_new_config_with_capability<DualAttestationLimit>(
+            account,
+            DualAttestationLimit { micro_lbr_limit: 1000000 },
+        );
+        move_to(tc_account, ModifyCapability { cap })
+    }
+
+    // Anyone can fetch the current set limit
+    public fun get_cur_microlibra_limit(): u64 {
+        let rule = LibraConfig::get<DualAttestationLimit>();
+        rule.micro_lbr_limit
+    }
+
+    public fun set_microlibra_limit(tc_account: &signer, micro_lbr_limit: u64) acquires ModifyCapability {
```

Comment:
I think if you publish `LibraConfig::ModifyConfigCapability<Self::DualAttestationLimit>` at the top-level of `tc_account`,  you can:
- delete `ModifyCapability`
- move the logic in this procedure to the `update_travel_rule_limit` script  (which can `move_from` the cap, pass a ref to `LibraConfig::set_with_capability`, and then move it back)

But this can also be done in a follow-up (I'm happy to do it); this looks great and has already gone through a lot of iteration!

Reply:
To clarify: publish the capability during genesis/tc account creation in `LibraAccount.move`, using the config_account signer?

Reply:
Yep!